server-api/data/storage.ts

```diff
@@ -0,0 +1,26 @@
+import { Units } from '../models/converter.model';
+
+const units: Units = {};
+
+units['b'] = {unit:'b', base: 'b', name: 'byte', type: 'storage', factor: 1, dataSource: "https://physics.nist.gov/cuu/Units/binary.html" };
```

Comment:
you are missing the a "bit" =)

Reply:
Ok, I added this in.  Let me know if there is anything else